Prace_domowe/Praca_domowa3/Jamrozy_Bartosz/pd3_australian_weather.ipynb

```diff
@@ -0,0 +1,493 @@
+{
```

Comment:
<p>Zabrakło krótkiego EDA aby wiedzieć czy nie należy dokonać inżynierii cech. W przypadku przeprowadzenia analiz okazałoby się, że zmienna Cloud3pm według dokumentacji powinna przyjmować wartości z zakresu 0-8, a raz w ramce danych pojawia się wartość 9.</p><p>Absolutne minimum to sprawdzenie balansu klas w zmiennej, która będzie targetem. Taka analiza implikuje czy accuracy jest dobrą miarą - dobrze, że na późniejszym etapie to zweryfikowano :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/164/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>